xmtp_mls/src/configuration.rs

```diff
@@ -0,0 +1,4 @@
+use openmls_traits::types::Ciphersuite;
+
+// TODO confirm ciphersuite choice
+pub const CIPHERSUITE: Ciphersuite = Ciphersuite::MLS_128_DHKEMX25519_AES128GCM_SHA256_Ed25519;
```

Comment:
I wonder if we actually want to just start with `MLS_128_DHKEMX25519_CHACHA20POLY1305_SHA256_Ed25519` given both are already implemented and we can choose whichever one we want.

Reply:
Happy to do this for now, we should still validate this choice though